app/src/main/java/com/goayo/debtify/model/Ledger.java

```diff
@@ -0,0 +1,13 @@
+package com.goayo.debtify.model;
+
+import java.util.List;
+
+public class Ledger {
+    public boolean createDebt(User lender, List<User> borrower, double owed) {
+        return false;
+    }
+
+    public boolean payOffDebt(double amount, String debtID) {
+        return true;
+    }
```

Comment:
Both team @gabrielbrattgard @yenanw  and team @OlofSjogren @Puh00 Created a ledger class.

Reply:
It's intentional, to make the JUnit test work. Otherwise compile error. Yenan and Gabriel's version of Ledger is final.

Reply:
nice

Reply:
Except we haven't done anything in the Ledger class, will implement later.